### DISCUSSION TITLE
Will it work with 1.12.2?

### DISCUSSION THREAD

**Discussion by Aqua-Alpha** [IrisShaders/Iris#964] [Q&A] [answered]
I’m trying to download a shader (BSL) without optifine. But the mod pack (Roguelike adventures and dungeons) is 1.12.2 version. And I don’t know if iris will work with it.

**Comment by maximumpower55** [comment, marked as answer]
no iris does not support 1.12.2 or forge read the readme please

**Comment by BeansBeefBroccoli** [comment]
No, Fabric doesn't support 1.12 and Iris doesn't support any modloader other than Fabric/Quilt.

**Comment by PumpkinXD** [comment]
nah
